### PR TITLE
ZENKO-1085 bugfix: backbeat liveness probes

### PR DESCRIPTION
**Pull request by giacope**
There is no point on having readiness probes for
the Backbeat pods since they don't serve requests
(except for BackbeatAPI).

Also, the **readiness** probe route is more appropriate for
the **liveness** probe since it detects when a connection
to a dependency (i.e. Zookeeper) is lost and
Kubernetes can take advantage of that to restart
pods and allow them to reconnect properly. This is
necessary since the Backbeat pods don't seem to recover
on their own from these disconnections.

### CODE REVIEW

File: kubernetes/zenko/charts/backbeat/values.yaml [hunk 1]
```diff
@@ -29,8 +29,7 @@ redis:
 health:
   port: 4042
   path:
-    liveness: /_/health/liveness
-    readiness: /_/health/readiness
+    liveness: /_/health/readiness
```
Comment (JianqinWang) [inline kubernetes/zenko/charts/backbeat/values.yaml:32]: I feel like there's a mismatch here

**Review by JianqinWang — CHANGES_REQUESTED** [review]

File: kubernetes/zenko/charts/backbeat/values.yaml (hunk 1 above)
Comment (giacope) [reply kubernetes/zenko/charts/backbeat/values.yaml:32]: @JianqinWang I will invite you to read the description 😅

Comment (JianqinWang) [reply kubernetes/zenko/charts/backbeat/values.yaml:32]: I see that, let me rephrase - as a whole I feel like this naming schema creates a naming inconsistency. However, if you feel this is the best way to resolve this and this kind of naming change can't be avoided, then I will approve.

**Review by JianqinWang — APPROVED** [review]

File: kubernetes/zenko/charts/backbeat/values.yaml (hunk 1 above)
Comment (giacope) [reply kubernetes/zenko/charts/backbeat/values.yaml:32]: It's clearly not the best way. But it's the only way of solving this issue without having to rebuild the `Backbeat` image (which we only do for releases). The reason for expediting this change without an image rebuild is because this is critical (IMHO).

Comment (JianqinWang) [reply kubernetes/zenko/charts/backbeat/values.yaml:32]: Thanks for the explanation

**Review by jonathan-gramain — APPROVED** [review]

**Comment by giacomoguiulfo** [comment]
Having these settings seem to cause quite a few number of restarts on some of the backbeat pods (because of liveness probe failures)
```
zenko-backbeat-gc-consumer-8486df8746-2hzf7                    1/1       Running   0          17h
zenko-backbeat-gc-consumer-8486df8746-mtvs2                    1/1       Running   1          16h
zenko-backbeat-gc-consumer-8486df8746-rg28h                    1/1       Running   0          17h
zenko-backbeat-gc-consumer-8486df8746-rph9t                    1/1       Running   0          17h
zenko-backbeat-gc-consumer-8486df8746-tfzs7                    1/1       Running   3          17h
zenko-backbeat-lifecycle-bucket-processor-8c7476df8-lz7nv      1/1       Running   12         17h
zenko-backbeat-lifecycle-bucket-processor-8c7476df8-nz6tz      1/1       Running   10         17h
zenko-backbeat-lifecycle-bucket-processor-8c7476df8-s5b54      1/1       Running   30         17h
zenko-backbeat-lifecycle-bucket-processor-8c7476df8-tkxbd      1/1       Running   32         16h
zenko-backbeat-lifecycle-bucket-processor-8c7476df8-z6pdc      1/1       Running   18         17h
zenko-backbeat-lifecycle-conductor-5c8b4458dc-57gwf            1/1       Running   28         17h
zenko-backbeat-lifecycle-object-processor-5969f6c8f6-44rxz     1/1       Running   19         17h
zenko-backbeat-lifecycle-object-processor-5969f6c8f6-4ndz8     1/1       Running   16         17h
zenko-backbeat-lifecycle-object-processor-5969f6c8f6-626mp     1/1       Running   36         16h
zenko-backbeat-lifecycle-object-processor-5969f6c8f6-ct7r7     1/1       Running   36         17h
zenko-backbeat-lifecycle-object-processor-5969f6c8f6-t9g6w     1/1       Running   11         17h
zenko-backbeat-replication-data-processor-59448d69cb-2k9g8     1/1       Running   4          17h
zenko-backbeat-replication-data-processor-59448d69cb-2rdlr     1/1       Running   3          17h
zenko-backbeat-replication-data-processor-59448d69cb-75rzw     1/1       Running   10         17h
zenko-backbeat-replication-data-processor-59448d69cb-9btf8     1/1       Running   5          17h
zenko-backbeat-replication-data-processor-59448d69cb-g46nh     1/1       Running   0          17h
zenko-backbeat-replication-data-processor-59448d69cb-gwdhd     1/1       Running   10         16h
zenko-backbeat-replication-data-processor-59448d69cb-qrzwd     1/1       Running   6          16h
zenko-backbeat-replication-data-processor-59448d69cb-rw86l     1/1       Running   0          17h
zenko-backbeat-replication-data-processor-59448d69cb-v44jg     1/1       Running   0          17h
zenko-backbeat-replication-data-processor-59448d69cb-z9z27     1/1       Running   1          17h
zenko-backbeat-replication-populator-7d7b5cf58d-v7p6g          1/1       Running   24         17h
zenko-backbeat-replication-status-processor-5c495587c6-645vq   1/1       Running   0          17h
zenko-backbeat-replication-status-processor-5c495587c6-klbhk   1/1       Running   0          17h
zenko-backbeat-replication-status-processor-5c495587c6-lnphz   1/1       Running   4          17h
zenko-backbeat-replication-status-processor-5c495587c6-vzj8b   1/1       Running   3          16h
zenko-backbeat-replication-status-processor-5c495587c6-xmk8w   1/1       Running   0          17h
```
Which is most likely caused because `mongodb`, `zenko-queue` and `zenko-quorum` some restarts:
```
zenko-mongodb-replicaset-0                                     2/2       Running   8          16h
zenko-mongodb-replicaset-1                                     2/2       Running   11         23h
zenko-mongodb-replicaset-2                                     2/2       Running   3          22h
zenko-mongodb-replicaset-3                                     2/2       Running   20         22h
zenko-mongodb-replicaset-4                                     2/2       Running   8          22h
zenko-zenko-queue-0                                            1/2       Running   4          23h
zenko-zenko-queue-1                                            2/2       Running   2          16h
zenko-zenko-queue-2                                            2/2       Running   8          22h
zenko-zenko-queue-3                                            2/2       Running   1          22h
zenko-zenko-queue-4                                            2/2       Running   2          22h
zenko-zenko-quorum-0                                           2/2       Running   1          23h
zenko-zenko-quorum-1                                           2/2       Running   1          22h
zenko-zenko-quorum-2                                           2/2       Running   8          22h
zenko-zenko-quorum-3                                           2/2       Running   0          16h
zenko-zenko-quorum-4                                           2/2       Running   0          22h
```


Any thoughts @jonathan-gramain, @rahulreddy?

**Review by tmacro — APPROVED** [review]
